plugins/woocommerce/includes/gateways/paypal/class-wc-gateway-paypal.php

```diff
@@ -145,6 +145,8 @@ public function __construct() {
 			add_filter( 'woocommerce_thankyou_order_received_text', array( $this, 'order_received_text' ), 10, 2 );
 		}
 
+		add_filter( 'woocommerce_settings_api_form_fields_paypal', array( $this, 'maybe_remove_fields' ), 15 );
```

Comment:
Should we rename the filter to `woocommerce_paypal_...`, consistent with how existing filters are name? e.g. `woocommerce_paypal_icon`, `woocommerce_paypal_force_one_line_item`

Reply:
Updated in https://github.com/woocommerce/woocommerce/pull/60082/commits/cbff3a551bdcf964b5bc1ee7242f3342fd054096

Reply:
This is an [existing WooCommerce filter](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/includes/abstracts/abstract-wc-settings-api.php#L67) (`woocommerce_settings_api_form_fields_<gateway_id>`), hence we can not change it.
But I have removed the `apply_filter` part here [5343394](https://github.com/woocommerce/woocommerce/pull/60082/commits/5343394850c89354529d6fa699dbfc5f798fe5e9) as it is unnecessary.